aerialist/px4/trajectory.py

```diff
@@ -245,7 +269,7 @@ def plot_multiple(
             fig.savefig(plot_file)
             plt.close(fig)
             if cls.WEBDAV_DIR is not None:
-                file_helper.upload(f"{cls.DIR}{filename}.png", cls.WEBDAV_DIR)
```

Comment:
cls.WEBDAV_DIR is set and used in so many palces. better to keep using the same variable for the moment